ricecooker/classes/files.py

```diff
@@ -356,14 +358,32 @@ class AudioFile(DownloadFile):
     def get_preset(self):
         return self.preset or format_presets.AUDIO
 
+
 class DocumentFile(DownloadFile):
     default_ext = file_formats.PDF
-    allowed_formats = [file_formats.PDF]
+    allowed_formats = [file_formats.PDF, file_formats.EPUB]
     is_primary = True
 
     def get_preset(self):
         return self.preset or format_presets.DOCUMENT
 
+
+class EPubFile(DownloadFile):
+    default_ext = file_formats.EPUB
+    allowed_formats = [file_formats.EPUB]  # ZEPUB other ???
+    is_primary = True
+
+    def get_preset(self):
+        return self.preset or format_presets.EPUB
+
+    def process_file(self):
```

Comment:
Was something going to be added here? If not, do we still need to implement the function here? (same with the validate function)